source/administration/config-settings.rst

```diff
@@ -3614,20 +3614,20 @@ Login Button Color
 Specify the color of the email login button for white labeling purposes. Use a hex code with a #-sign before the code.
 
 +-------------------------------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"LoginButtonColor": ""`` with string input.                                       |
+| This feature's ``config.json`` setting is ``"LoginButtonColor": ""`` with string input. This setting only applies to the mobile apps.                                      |
```

Comment:
Actually, did you mean to put this on line 3614 above?

Reply:
Whoops, yes! Fixed